app/src/main/java/com/example/keniphhomemaintenance/UtilitiesScreen.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.keniphhomemaintenance
```

Comment:
UtilitiesScreen.kt is a placeholder for Phil's screen; replace this with Phil's work.
Should talk about passing arguments / keeping this screen tied to the respective dwelling first